bin/phase-helpers.sh

```diff
@@ -360,14 +360,33 @@ unpack() {
 		fi
 		[[ ! -s ${srcdir}${x} ]] && die "unpack: ${x} does not exist"
 
+		suffix_known=""
+		case ${suffix_insensitive} in
+			tar|tgz|tbz|tbz2|zip|jar|gz|z|bz2|bz|a|deb|lzma) suffix_known=1 ;;
+			7z)      ___eapi_unpack_supports_7z  && suffix_known=1 ;;
+			rar)     ___eapi_unpack_supports_rar && suffix_known=1 ;;
+			lha|lzh) ___eapi_unpack_supports_lha && suffix_known=1 ;;
+			xz)      ___eapi_unpack_supports_xz  && suffix_known=1 ;;
+			txz)     ___eapi_unpack_supports_txz && suffix_known=1 ;;
+		esac
+
+		if ___eapi_unpack_is_case_sensitive \
+				&& [[ ${suffix} != "${suffix_insensitive}" ]] \
+				&& ! has ${suffix} ZIP Z 7Z RAR LHA LHa; then
```

Comment:
```suggestion
				&& ! has "${suffix}" ZIP Z 7Z RAR LHA LHa "${suffix_insensitive}"; then
```

should be the same, right? Also note that technically `suffix` can contain spaces.

Or perhaps we could even make this simpler and do something like:

```
___eapi_unpack_is_case_sensitive || suffix=${suffix,,}
```

Reply:
Good catch, fixed.